includes/Core/Authentication/Authentication.php

```diff
@@ -362,6 +373,17 @@ public function verification_file() {
 		return $this->verification_file;
 	}
 
+	/**
+	 * Gets the connected_admins instance.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return HasConnectedAdmins ConnectedAdmins instance.
+	 */
+	public function connected_admins() {
+		return $this->connected_admins;
+	}
```

Comment:
I think we can omit the addition of this method for now. There are a few instances on `Authentication` that are not exposed in this way (e.g. `First_Admin`), and this class is similar. Let's not add this until we need it 👍

Reply:
Ok, removed.